tools/BaseRunner.py

```diff
@@ -45,10 +44,12 @@ def run(self, tmp_dir, params):
         """
         self.prepare_run_cb(tmp_dir, params)
 
-        proc = subprocess.Popen(self.cmd, cwd=tmp_dir,
-                                preexec_fn=set_process_limits,
-                                stdout=subprocess.PIPE,
-                                stderr=subprocess.STDOUT)
+        proc = subprocess.Popen(
```

Comment:
Can we configure it to leave 'arguments start after open parenthesis and are indended underneath' alone ? It makes for much more readable code.
The 'newline after open parenthesis' should only really be used when the arguments get too long. In any case, that should be the programmers decision.

We should make sure that formatters fix readability where needed, but don't obliterate where not needed.

Reply:
Please leave the settings alone.

Reply:
My suggestions is towards readability,  wasting less vertical space and being in accordance to Google Python style-guide that I'd usually use as reference.

But if it is preferred this way in Symbiflow I won't object.